src/utils/viewer/viewer.cpp

```diff
@@ -552,11 +588,90 @@ void Viewer::viewportEvent(ViewportEvent& event) {
   renderCamera_->setViewport(event.windowSize());
   imgui_.relayout(Mn::Vector2{event.windowSize()} / event.dpiScaling(),
                   event.windowSize(), event.framebufferSize());
+
+  /* Recreate object ID reading renderbuffers that depend on viewport size */
+  selectionDepth_ = Mn::GL::Renderbuffer{};
```

Comment:
Some duplicate code here from around L333. It'd be nice to see a helper function instead.